documentation for the Minut Point (#7269)
home-assistant/home-assistant.github.io: source/_components/point.markdown

---
layout: page
title: "Minut Point"
description: "Instructions on how to integrate Minut Point into Home Assistant."
date: 2018-10-30
sidebar: true
comments: false
sharing: true
footer: true
logo: minut.svg
ha_category: Hub
featured: true
ha_iot_class: "Cloud Polling"
---

The Point component is the main component to integrate the [Minut Point](https://minut.se/). To connect Point, you will have to [sign up for a developer account](https://minut.com/community/developers/) and get a `client_id` and `client_secret` using the `callback url` shown in the integration flow.

<p class='note'>

> The Point is just active occasionaly so the sensors are only updated every hour or so.

Reviewer: ```suggestion
The Point is just active occasionally so the sensors are only updated every hour or so.
```

The events sent from the Point is sent as a webhook back to Home Assistant with `event_type` as `point_webhook_received`, please consider the documentation for the [IFTT](https://www.home-assistant.io/components/ifttt/) component on how to write automations for webhooks.
</p>

### {% linkable_title Configuration %}

Integration to Point is configured via the Integrations setting found in the Configuration section on your Home Assitant page.

> Just follow the integration dialogs and make sure that your Home Assistant instance is accessable from the web (to be able to recieve webhooks).

Reviewer: ```suggestion
Just follow the integration dialogs and make sure that your Home Assistant instance is accessible from the web (to be able to receive webhooks).
```

---
layout: page
title: "Minut Point"
description: "Instructions on how to integrate Minut Point into Home Assistant."
date: 2018-10-30
sidebar: true
comments: false
sharing: true
footer: true
logo: minut.svg
ha_category: Hub
featured: true
ha_iot_class: "Cloud Polling"

> ---

Reviewer 2: `ha_release` is missing.

---
layout: page
title: "Minut Point"
description: "Instructions on how to integrate Minut Point into Home Assistant."
date: 2018-11-19
sidebar: true
comments: false
sharing: true
footer: true
logo: minut.svg
ha_category: Hub
featured: true
ha_release: 0.83
ha_iot_class: "Cloud Polling"
---

The Point component is the main component to integrate the [Minut Point](https://minut.se/). To connect Point, you will have to [sign up for a developer account](https://minut.com/community/developers/) and get a `client_id` and `client_secret` using the `callback url`  as `base_url` + `/api/minut` eg. `http://localhost:8123/api/minut`. The `client_id` and `client_secret` should be used as below. 

> Once Home Assistant is started, a configurator will pop up asking you to Authenticate your Point account via a link when you follow the link and have clicked on **Accept** you will be redirected to the `callback url` and the Point integration will be automatically configures and you can go back to the original dialog and press **Submit**.

Reviewer 2: ```suggestion
Once Home Assistant is started, a configurator will pop up asking you to Authenticate your Point account via a link when you follow the link and have clicked on **Accept** you will be redirected to the `callback url` and the Point integration will be automatically configured and you can go back to the original dialog and press **Submit**.
```

### {% linkable_title Configuration %}

```yaml
# Example configuration.yaml entry
point:
  client_id: CLIENT_ID
  client_secret: CLIENT_SECRET
```

> 

Reviewer 2: Configuration options list is missing.

Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration
